tests/system/marathon_common_tests.py

```diff
@@ -1133,3 +1133,31 @@ def http_output_check():
         assert 'Relay from {}'.format(relay_app["id"]) in output
 
     http_output_check()
+
+
+@shakedown.dcos_1_11
+def test_ipv6_healthcheck():
+    """ There is new feature in DC/OS 1.11 that allows containers running on IPv6 network to be healthchecked from
+        Marathon. This tests verifies executing such healthcheck.
+    """
+    agents = shakedown.get_agents()
+    network_cmd = "sudo docker network create --driver=bridge --ipv6 --subnet=fd01::/64 mesos-docker-ipv6-test"
```

Comment:
What is this doing?

Reply:
to be able to run container on ipv6 I have to have ipv6 network. This creates the network. Then in the app json you can see that the container I am launching uses this network...

I'll add comment there